pkg/karmadactl/get.go

```diff
@@ -961,8 +943,8 @@ func (g *CommandGetOptions) getRBInKarmada(gclient karmadaclientset.Interface) e
 	return nil
 }
 
-// getClusterInKarmadaForClient get cluster info in karmada cluster
-func getClusterInKarmadaForClient(gclient karmadaclientset.Interface) (map[string]*ClusterInfo, error) {
+// GetClusterInKarmada get cluster info in karmada cluster
+func GetClusterInKarmada(gclient karmadaclientset.Interface) (map[string]*ClusterInfo, error) {
```

Comment:
Why we need the function and the ClusterInfo struct. 

we can use directly  karmadaclientset.Interface to get cluster list in the Run function

Reply:
Yes, just looked it, we don't need this function and struct now.